lib/cli/options.ts

```diff
@@ -42,11 +41,11 @@ export class Option {
     if (value && typeof value === 'string') {
       return '' + value;
     }
-    return null;
+    return '';
```

Comment:
[Here's the line](https://github.com/boneskull/webdriver-manager/blob/88d6105f538f075968c152935131bf19bf289532/lib/cmds/update.ts#L88-L88) that prompted the change.